rest_sensors.cpp

```diff
@@ -1039,6 +1012,8 @@ int DeRestPluginPrivate::changeSensorConfig(const ApiRequest &req, ApiResponse &
                 }
                 else if (rid.suffix == RConfigOffset) // Signed integer
                 {
+                    offset = data.integer - item->toNumber();
```

Comment:
Wouldn't `offset = data.integer` be more adequate here?

Reply:
No, you need to undo the previous offset value and then apply the new offset.  The temperature/humidiy ResourceItem holds the adjusted value, not the value reported by the sensor.